lightningd/test/run-invoice-select-inchan.c

```diff
@@ -287,6 +287,10 @@ void notify_connect(struct lightningd *ld UNNEEDED, struct node_id *nodeid UNNEE
 /* Generated stub for notify_disconnect */
 void notify_disconnect(struct lightningd *ld UNNEEDED, struct node_id *nodeid UNNEEDED)
 { fprintf(stderr, "notify_disconnect called!\n"); abort(); }
+/* Generated stub for notify_invoice_payment */
+void notify_invoice_payment(struct lightningd *ld UNNEEDED, struct amount_msat amount UNNEEDED,
+						    struct preimage preimage UNNEEDED, const struct json_escape *label UNNEEDED)
```

Comment:
And here.

Reply:
As indicated by the comment above, I did not write that.